src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/Worker.java

```diff
@@ -1282,6 +1322,10 @@ public Worker build() {
                     .getProxy(config.getStreamName());
             }
 
+            if (workerStateChangeListener == null){
```

Comment:
Please match the code style and add a space after the \)

Reply:
sorry :(. Will do :)